Cleanup E0412 and E0422 (#69917)
rust-lang/rust: src/librustc_error_codes/error_codes/E0422.md

> An identifier that is either undefined or not a struct was used.

Reviewer: ```suggestion
An identifier that is neither undefined nor a struct was used.
```
